morse_decoder.rb

```diff
@@ -0,0 +1,29 @@
+module MorseCodeDecoder
+  MORSE_CODE_DICT = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C',
+    '-..' => 'D', '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H', '..' => 'I',
+    '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T', '..-' => 'U',
+    '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }.freeze
+
+  def self.decode_char(morse_char)
+    MORSE_CODE_DICT.key(morse_char)
+  end
```

Comment:
- [ ] Nice work saving the characters in a constant variable :+1: However the  method `decode_char` for decoding a single character is returning `nil`. Kindly make sure the method return the value of key entered. 
**HIT:** `MORSE_CODE_DICT[morse_char]`